README.md

```diff
@@ -128,7 +136,7 @@ Make sure that your `$GOPATH/bin` is in your `$PATH`.
    
    It will generate a reverse proxy `path/to/your_service.pb.gw.go`.
 
-   Note: After generating the code for each of the stubs, in order to build the code, you will want to run ```go get .``` from the directory containing the stubs.
+   Note: After generating the code for each of the stubs, in order to build the code, you will want to move the stubs into your `$GOPATH/src` working directory and run ```go get .``` from the directory containing the stubs.
```

Comment:
the protoc command creates the stubs in the same directory as the service.proto file. is it possible to provide an output directory for the protoc plugins?